src/setup.py

```diff
@@ -46,7 +46,7 @@ def read(fname):
         'knack==0.1.1',
         'msrest>=0.4.4',
         'requests',
-        'azure-servicefabric==6.0',
+        'azure-servicefabric==6.0.2',
```

Comment:
CI is not passing

Reply:
I've ran the travis script locally in a virtualenv and can repeat the failure. `pip install adal` will resolve the issue - do you want me to add 'adal' to the requirements.txt or should we fix 6.0.2 dependency graph to ensure the package is pulled in?

Reply:
If it's a dependency of version 6.0.2 of `azure-servicefabric` then the requirements for the package must be updated. I don't think your changes here require adal explicitly.

Reply:
It seems that between version 6.0.0 and 6.0.1 of azure-servicefabric that the msrestazure module is not being pulled in (although it still remains referenced in the requirements.txt) and therefore adal (which is a dependency of msrestazure) is not pulled in either. There is an explicit reference to 'adal' in src/sfctl/auth.py therefore I think it makes sense to put it in sfctl's requirements.txt rather than rely on an external package's dependency - I'm no expert in python dependencies so happy to take your advice on this.

Reply:
Yeah it's fine to add it as a dependency for sfctl. I must have missed that whoever authored the auth changes didn't add that package.